File: ronald036/programacao_basica/questao5.java
import java.util.Scanner;

public class questao5 {
   public static void main(String[] args) {
	int simples;
	int completo;
	int lucro;
	Scanner k;
	k = new Scanner (System.in);
	
	System.out.println("** seguradora xyz **");
	System.out.println("     >> Sistema de avaliação <<");
	
	System.out.print("quantidade de contratos simples : ");
	simples = k.nextInt();

	System.out.print("quantidade de contratos completos : ");
	completo = k.nextInt();

	lucro = simples * 2300 + completo * 3500;

	System.out.println("lucro obtido: R$ " + (lucro));	
	
	}}

File: ronald036/programacao_basica/questao5.py
print("** Seguradora **")
print("    >> Sistema de avaliação <<")
qtd_contratos_completos = int(input('qtd de contratos completos: '))
qtd_contratos_simples = int(input('qtd de contratos simples: '))
total = qtd_contratos_completos * 350 + qtd_contratos_simples * 230
print('total: R$' + str(total))
